js/chatbot/shared/Message.svelte

```diff
@@ -131,30 +138,52 @@
 						message.content.component === "html"}
 					class:thought={thought_index > 0}
 				>
-					<button
-						data-testid={role}
-						class:latest={i === value.length - 1}
-						class:message-markdown-disabled={!render_markdown}
-						style:user-select="text"
-						class:selectable
-						style:cursor={selectable ? "pointer" : "default"}
-						style:text-align={rtl ? "right" : "left"}
-						on:click={() => handle_select(i, message)}
-						on:keydown={(e) => {
-							if (e.key === "Enter") {
-								handle_select(i, message);
-							}
-						}}
-						dir={rtl ? "rtl" : "ltr"}
-						aria-label={role + "'s message: " + get_message_label_data(message)}
-					>
-						{#if message.type === "text"}
-							<div class="message-content">
-								{#if message?.metadata?.title}
-									<MessageBox
-										title={message.metadata.title}
-										expanded={is_last_bot_message([message], value)}
-									>
+					{#if in_edit_mode && thought_index === messages.length - 1 && message.type === "text"}
+						<!-- svelte-ignore a11y-autofocus -->
+						<textarea
```

Comment:
Is it possible to preserve the original message width? Or at least bump it up as it's hard now to edit longer messages

![edit_width_display](https://github.com/user-attachments/assets/3d0bbc08-17ae-431e-acb5-05b3f58db0b6)

Reply:
Would also help with longer messages that include code blocks

Reply:
![Recording 2024-12-18 at 15 50 16](https://github.com/user-attachments/assets/96a88820-4e0f-4da6-a0f0-bb5e9d9b50ba)

done